billing.go

```diff
@@ -59,18 +55,35 @@ func (c *Client) CreateBillingPlan(plan BillingPlan) (*CreateBillingResp, error)
 	return response, err
 }
 
+// UpdateBillingPlan updates values inside a billing plan
+// Endpoint: PATCH /v1/payments/billing-plans
+func (c *Client) UpdateBillingPlan(planId string, pathValues map[string]map[string]interface{}) error {
+	patchData := []Patch{}
+	for path, data := range pathValues {
+		patchData = append(patchData, Patch{
+			Operation: "replace",
+			Path:      path,
+			Value:     data,
+		})
+	}
+
+	jsonData, err := json.Marshal(patchData)
+	buf := bytes.NewBuffer(jsonData)
+	req, err := c.NewRequest(http.MethodPatch, fmt.Sprintf("%s%s%s", c.APIBase, "/v1/payments/billing-plans/", planId), buf)
+	if err != nil {
+		return err
+	}
+	err = c.SendWithAuth(req, nil)
+	return err
+}
+
 // ActivatePlan activates a billing plan
 // By default, a new plan is not activated
 // Endpoint: PATCH /v1/payments/billing-plans/
 func (c *Client) ActivatePlan(planID string) error {
-	buf := bytes.NewBuffer([]byte(`[{"op":"replace","path":"/","value":{"state":"ACTIVE"}}]`))
-	req, err := http.NewRequest("PATCH", fmt.Sprintf("%s%s", c.APIBase, "/v1/payments/billing-plans/"+planID), buf)
-	if err != nil {
-		return err
-	}
-	req.SetBasicAuth(c.ClientID, c.Secret)
-	req.Header.Set("Authorization", "Bearer "+c.Token.Token)
-	return c.SendWithAuth(req, nil)
+	return c.UpdateBillingPlan(planID, map[string]map[string]interface{}{
+		"/": {"state": "active"},
```

Comment:
could you update this to use your `ACTIVE` const above. seems like that is what is needed.

https://developer.paypal.com/docs/api/payments.billing-plans/v1/#billing-plans_patch